streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateUpdateTask.java

```diff
@@ -69,7 +72,8 @@ public GlobalStateUpdateTask(final ProcessorTopology topology,
                     source,
                     deserializationExceptionHandler,
                     logContext,
-                    processorContext.metrics().skippedRecordsSensor()
+                    // task-id would be "-1_-1"
+                    processorContext.metrics().taskLevelSensor(SKIPPED_RECORDS, processorContext.taskId().toString(), Sensor.RecordingLevel.INFO)
```

Comment:
We need a task-id for the global-state-update-task for its skipped-records metrics.